projects/npm-tools/packages/npm-scripts/src/utils/getMergedConfig.js

```diff
@@ -130,6 +130,16 @@ function getMergedConfig(type, property) {
 			]);
 			break;
 
+		case 'bundler-with-federation':
+			mergedConfig = deepMerge([
+				require('../config/npm-bundler'),
+				getUserConfig('npmbundler'),
+				{
+					ignore: ['webpack/**/*'],
+				},
```

Comment:
The order here seems a bit off compared to every other config type that we merge: ie. "our base stuff" → "user stuff" → "more of our stuff" (vs all the others, which let the user have the last say).

Could we just fold the "more of our stuff" into `../config/npm-bundler`? I know a user is never going to want to override this, but just for consistency... and the `ignore` would be harmless as far as I can tell the `case 'bundler'` block (because nobody is writing to `webpack/**/*`).

Reply:
Yeah, maybe that would simplify the code too. My first thought was that someone would override that ignore accidentally, but seeing it from the other side, if someone overrides it is because they are using `webpack` folder for something and that will surely lead to some kind of error (because of the conflict).

Maybe this is a sign that `webpack` is not a good choice for the name of this internal folder. Perhaps we should change it to something more internal. Like `_liferay_webpack` or something that denotes its technical nature. 

What do you think?

Reply:
> Maybe this is a sign that `webpack` is not a good choice for the name of this internal folder. Perhaps we should change it to something more internal. Like `_liferay_webpack` or something that denotes its technical nature.

Sounds great. I was thinking that too, but didn't say it because I have already said too much.

A convention I've used elsewhere is `__generated__/` — which is a pretty obvious cue not to go putting anything hand-created in there, and translates to any context where we use tool-generated files. But really, any name would be less collision-prone than `webpack`.